test/functionalTest/cases/000_ngsi10_convenience/attribute_value_instance.test

```diff
@@ -155,7 +155,7 @@ Date: REGEX(.*)
   <statusCode>
     <code>404</code>
     <reasonPhrase>No context element found</reasonPhrase>
-    <details>Entity-Attribute pair: /E01-A1/</details>
+    <details>Entity id: /E01/</details>
```

Comment:
I know.
This is how it came out.
I didn't care to spend time to see why, for this request that we don't even want ... :-)

If you find it important, I'll try to fix it to return the same response as before.